src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -68,8 +68,8 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
           var expectedSchema =
             StructType(Seq(StructField("RGUID", StringType), StructField("Date", StringType)))
           if (enableLineage) {
-            expectedSchema =
-              expectedSchema.add(StructField(IndexConstants.DATA_FILE_NAME_COLUMN, StringType))
+            expectedSchema = expectedSchema.add(
+              StructField(IndexConstants.DATA_FILE_NAME_COLUMN, StringType, false))
```

Comment:
Btw, looks like we don't have a coverage of checking the content of index since there was no change in tests related to this PR? Shall we add a simple one in `CreateIndexTests.scala`?

Reply:
Sure, I added a new unit test case in `CreateIndexTests.scala`, named "Verify content of lineage column."